client/src/components/Intro/index.js

```diff
@@ -72,13 +73,17 @@ function Intro({
           </Col>
         </Row>
         <Row>
-          <Col sm={10} smOffset={1} xs={12}>
-            <Spacer />
-            <h4>
-              If you are new to coding, we recommend you{' '}
-              <Link to={slug}>start at the beginning</Link>.
-            </h4>
-          </Col>
+          {completedChallengeCount < 15 ? (
+            <Col sm={10} smOffset={1} xs={12}>
+              <Spacer />
+              <h4>
+                If you are new to coding, we recommend you{' '}
+                <Link to={slug}>start at the beginning</Link>.
+              </h4>
+            </Col>
+          ) : (
+            ''
```

Comment:
In the issue #38781 @ahmadabdolsaheb stated [this](https://github.com/freeCodeCamp/freeCodeCamp/issues/38781#issue-615375889).

He wanted these two as a separate feature if I am not misunderstanding....😅

He said that-

>remove "If you are new to coding, we recommend you start at the beginning." message if the user has completed a certain number of challenges.

So, should I update the PR or is this enough @ojeytonwilliams ?

Reply:
Sorry, I missed that discussion.  It looks to me like the final decision hasn't been made yet.  Given that, please don't implement what I've suggested until that decision's been clarified, as it may not be what's wanted.

Reply:
Logic states that:

For a signed in user:

1. The start here action should be when a user has less completed challenges than a certain threshold no. of challenges.
2. There should be a go to a current challenge action (a button in the stack is a good idea)

For a non-signed in user:

1. The start here action should be there always, because we do not save progress.
2. The current challenge action should not be there, because we would want them to click the "sign in and save your progress" action instead.

Reply:
The stack currently being "View my Portfolio", "Update my account settings"?.

Reply:
> The stack currently being "View my Portfolio", "Update my account settings"?.

Yes.

Reply:
Thanks a lot clarifying guys! Much appreciated!